net/ola/Portfile

```diff
@@ -4,8 +4,7 @@ PortSystem          1.0
 PortGroup           github 1.0
 PortGroup           cxx11 1.1
 
-github.setup        OpenLightingProject ola 0.10.7
-revision            2
```

Comment:
Please restore the `revision` line and set it back to 0.

Reply:
Sorry, I think the previous time I'd reset I just removed it.